src/Taylor1.jl

```diff
@@ -666,3 +669,53 @@ function deriv{T<:Number}(a::Taylor1{T}, n::Int=1)
     @assert a.order >= n >= 0
     factorial( widen(n) ) * a.coeffs[n+1] :: T
 end
+
+
+"""
+    A_mul_B!(y, a, b)
+
+Multiply a*b and save the result in y.  Extends y to fit the results if necessary.
+"""
+function A_mul_B!{T<:Number}(y::Vector{Taylor1{T}},a::Matrix{T},b::Vector{Taylor1{T}})
+
+    # determine the maximal order of b
+    order = maximum([b1.order for b1 in b])
+
+    n,k = size(a)
+
+    # extend y to fit all the coefficients and zero the
```

Comment:
I think we may (and should) assume that `y` is already defined, though we need to ensure that it has the proper dimensionality (length `n`). Otherwise, we may run into an error. As an example, using your tests, but changing the dimensionality of Y to something less than 90 yields a `BoundsError`.